compiler/control/OMROptions.cpp

```diff
@@ -1458,12 +1465,17 @@ OMR::Options::setString(char *option, void *base, TR::OptionTable *entry)
    return dummy_string;
    }
 
+char *
+OMR::Options::setStringInJitConfig(char *option, void *base, TR::OptionTable *entry)
```

Comment:
I have added Doxygen-style comments. Same as above, 'base' is an-used parameter that needs to be present due to the fixed format of the option processing routines.